src/Tools/ExternalAccess/Razor/InternalAPI.Unshipped.txt

```diff
@@ -110,7 +110,7 @@ Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorMappedSpanResult.RazorMappedSpa
 Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorMappedSpanResult.RazorMappedSpanResult(string! filePath, Microsoft.CodeAnalysis.Text.LinePositionSpan linePositionSpan, Microsoft.CodeAnalysis.Text.TextSpan span) -> void
 Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorPinnedSolutionInfoWrapper
 Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorPinnedSolutionInfoWrapper.RazorPinnedSolutionInfoWrapper() -> void
-Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorPinnedSolutionInfoWrapper.RazorPinnedSolutionInfoWrapper(Microsoft.CodeAnalysis.Checksum! underlyingObject) -> void
+Microsoft.CodeAnalysis.ExternalAccess.Razor.RazorPinnedSolutionInfoWrapper.RazorPinnedSolutionInfoWrapper(Microsoft.CodeAnalysis.Checksum underlyingObject) -> void
```

Comment:
`!` versus nothing is no difference at the binary level.  `!` just meant that when this was a reference type, you woudn't get null.  But as a value type, it's implicitly `!` since it can't ever be null.